docs/content/features/run_on_aws.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Infection Monkey AWS Execution"
+draft: false
+description: "Run Infection Monkey on AWS EC2 instances"
+tags: ["aws", "ec2", "ssm", "agent", "run"]
+pre: "<i class='fa-brands fa-aws'></i> "
+---
+
+## Description
+
+The Infection Monkey can run on AWS EC2 instances using the
+[AWS Systems Manager (SSM) Agent](
+https://docs.aws.amazon.com/systems-manager/latest/userguide/ssm-agent.html).
+Each EC2 instance has an SSM Agent installed for updating, managing, and
+configuring the instance. It executes requests received from
+the [AWS Systems Manager](
+https://docs.aws.amazon.com/systems-manager/latest/userguide/what-is-systems-manager.html),
+and uses the [Amazon Message Delivery Service](
+https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonmessagedeliveryservice.html)
+to report back status and execution details.
+
+Infection Monkey AWS Execution lets you run Infection Monkey Agent on your
+chosen EC2 instances without the requirement of manual installation, enabling
+the simulation of various attack scenarios across your AWS infrastructure.
+
+## Prerequisites
+
+In order to ensure that Infection Monkey can be executed on EC2 instance,
+the SSM Agent must be present on the target instance. The list of AWS AMIs
+that come with the SSM Agent pre-installed can be found [here](
+https://docs.aws.amazon.com/systems-manager/latest/userguide/ami-preinstalled-agent.html).
```

Comment:
"The list of AWS AMIs that come with the SSM Agent pre-installed..."

This isn't relevant to Infection Monkey. We can get rid of it.

Reply:
It is relevant for where this feature can run on.

Reply:
Saying the SSM Agent needs to be on the system is enough information wrt Infection Monkey. Which instances have it pre-installed is solely an AWS thing which the users can look up themselves if they need to. It's out of place here.